packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -312,36 +351,40 @@ you access the appropriate property of the interface returned from `variables`,
 which represents a single variable.
 Example:
 
-```ts
-// MyAction is some action type that produces variables
+```ts fixture=action
+// MyAction is some action type that produces variables, like EcrSourceAction
 const myAction = new MyAction({
   // ...
+  actionName: 'myAction',
 });
 new OtherAction({
   // ...
   config: myAction.variables.myVariable,
+  actionName: 'otherAction',
 });
 ```
```

Comment:
I created a fixture that basically holds the dummy `MyAction` and `OtherAction` classes. Hopefully that's the right way to go here...

Reply:
Sure! You could also have stuck them in `default` and that would have been fine too, but this is good.